app/src/main/java/com/example/una/RecyclerViewModels/HorizontalModel.java

```diff
@@ -0,0 +1,21 @@
+package com.example.una.RecyclerViewModels;
+
+public class HorizontalModel {
+    String name, description;
```

Comment:
Declare these on different lines so it will be easier to review future code that adds any fields here